lib/payment_services/crypto_apis/payout_adapter.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+require_relative 'payout_client'
+require_relative 'wallet'
+require_relative 'payout_payment'
+require_relative 'payout'
+
+class PaymentServices::CryptoApis
+  class PayoutAdapter
+    # payout_wallets - хеш из кошельков(кошелька) для снятия средств с уже просчитанными суммами в формате:
+    # { wallet_id => payout_amount }
+    def initialize(order:, payout_wallets:)
+      @order = order
+      @payout_wallets = payout_wallets
+    end
+
+    attr_reader :order, :payout_wallets
+
+    def create_payout(amount, destination_address, fee)
+      payout = Payout.new(amount: amount, order_public_id: order.public_id, destination_address: destination_address, fee: fee)
```

Comment:
Or we could use `recommended` fee from [Get Transactions Fee](https://docs.cryptoapis.io/rest-apis/blockchain-as-a-service-apis/bch/index#bch-hd-wallet-transactions-size) endpoint to refuse `fee` dependency.